<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NoiseReductionWidget</class>
 <widget class="QWidget" name="NoiseReductionWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>255</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabs_noiseReduction">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_GREYC">
      <attribute name="title">
       <string>GREYCStoration</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="tabs_GREYC">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_regularization">
          <attribute name="title">
           <string>Regularization</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="0" column="0" colspan="2">
              <widget class="QCheckBox" name="checkBox_regularizationEnabled">
               <property name="enabled">
                <bool>true</bool>
               </property>
               <property name="toolTip">
                <string>Enable GREYCStoration Noise Reduction Filter</string>
               </property>
               <property name="text">
                <string>Enabled</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="3" colspan="2">
              <widget class="QCheckBox" name="checkBox_fastApproximation">
               <property name="toolTip">
                <string>Use Fast Approximation</string>
               </property>
               <property name="text">
                <string>Fast approximation</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_iterations">
               <property name="text">
                <string>Iterations</string>
               </property>
               <property name="buddy">
                <cstring>slider_iterations</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="1" colspan="3">
              <widget class="QSlider" name="slider_iterations">
               <property name="toolTip">
                <string>Adjust number of Iterations</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="1" column="4">
              <widget class="QDoubleSpinBox" name="spinBox_iterations">
               <property name="toolTip">
                <string>Adjust number of Iterations Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>16.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_amplitude">
               <property name="text">
                <string>Amplitude</string>
               </property>
               <property name="buddy">
                <cstring>slider_amplitude</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1" colspan="3">
              <widget class="QSlider" name="slider_amplitude">
               <property name="toolTip">
                <string>Adjust Filter Strength/Amplitude</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="4">
              <widget class="QDoubleSpinBox" name="spinBox_amplitude">
               <property name="toolTip">
                <string>Adjust Filter Strength/Amplitude Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>200.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>5.000000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_precission">
               <property name="text">
                <string>Precision</string>
               </property>
               <property name="buddy">
                <cstring>slider_precision</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1" colspan="3">
              <widget class="QSlider" name="slider_precision">
               <property name="toolTip">
                <string>Adjust precision of Gaussian Filter Kernel</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="4">
              <widget class="QDoubleSpinBox" name="spinBox_precision">
               <property name="toolTip">
                <string>Adjust precision of Gaussian Filter Kernel Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_adv">
          <attribute name="title">
           <string>Adv</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="0" column="0" colspan="2">
              <widget class="QGroupBox" name="groupBox_integration">
               <property name="title">
                <string>Integration</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_angular">
                    <property name="text">
                     <string>Spatial</string>
                    </property>
                    <property name="buddy">
                     <cstring>slider_spatial</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QSlider" name="slider_spatial">
                    <property name="toolTip">
                     <string>Amount of Spatial Integration</string>
                    </property>
                    <property name="maximum">
                     <number>512</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QDoubleSpinBox" name="spinBox_spatial">
                    <property name="toolTip">
                     <string>Amount of Spatial Integration Value</string>
                    </property>
                    <property name="keyboardTracking">
                     <bool>false</bool>
                    </property>
                    <property name="decimals">
                     <number>3</number>
                    </property>
                    <property name="maximum">
                     <double>1.000000000000000</double>
                    </property>
                    <property name="singleStep">
                     <double>0.050000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_spatial">
                    <property name="text">
                     <string>Angular</string>
                    </property>
                    <property name="buddy">
                     <cstring>slider_angular</cstring>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QSlider" name="slider_angular">
                    <property name="toolTip">
                     <string>Amount of Angular Integration</string>
                    </property>
                    <property name="maximum">
                     <number>512</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QDoubleSpinBox" name="spinBox_angular">
                    <property name="toolTip">
                     <string>Amount of Angular Integration Value</string>
                    </property>
                    <property name="keyboardTracking">
                     <bool>false</bool>
                    </property>
                    <property name="decimals">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <double>90.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_interplType">
               <property name="text">
                <string>Interpolation type:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QComboBox" name="comboBox_interpolType">
               <property name="toolTip">
                <string>Choose Interpolation Kernel Type</string>
               </property>
               <item>
                <property name="text">
                 <string>Nearest-Neighbour</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Linear</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Runge-Kutta</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_11">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_filter">
          <attribute name="title">
           <string>Filter</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="0" column="0">
              <widget class="QLabel" name="label_alpha">
               <property name="text">
                <string>Alpha</string>
               </property>
               <property name="buddy">
                <cstring>slider_alpha</cstring>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QSlider" name="slider_alpha">
               <property name="toolTip">
                <string>Adjust Gaussian Filter Alpha</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QDoubleSpinBox" name="spinBox_alpha">
               <property name="toolTip">
                <string>Adjust Gaussian Filter Alpha Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_sigma">
               <property name="text">
                <string>Sigma</string>
               </property>
               <property name="buddy">
                <cstring>slider_sigma</cstring>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QDoubleSpinBox" name="spinBox_sigma">
               <property name="toolTip">
                <string>Adjust Gaussian Filter Sigma Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>12.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.500000000000000</double>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_sharpness">
               <property name="text">
                <string>Sharpness</string>
               </property>
               <property name="buddy">
                <cstring>slider_sharpness</cstring>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSlider" name="slider_sharpness">
               <property name="toolTip">
                <string>Adjust Sharpness</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QDoubleSpinBox" name="spinBox_sharpness">
               <property name="toolTip">
                <string>Adjust Sharpness Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="maximum">
                <double>2.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_aniso">
               <property name="text">
                <string>Aniso</string>
               </property>
               <property name="buddy">
                <cstring>slider_aniso</cstring>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QSlider" name="slider_aniso">
               <property name="toolTip">
                <string>Adjust Filter Anisotropy</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QDoubleSpinBox" name="spinBox_aniso">
               <property name="toolTip">
                <string>Adjust Filter Anisotropy Value</string>
               </property>
               <property name="keyboardTracking">
                <bool>false</bool>
               </property>
               <property name="decimals">
                <number>3</number>
               </property>
               <property name="maximum">
                <double>1.000000000000000</double>
               </property>
               <property name="singleStep">
                <double>0.050000000000000</double>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSlider" name="slider_sigma">
               <property name="toolTip">
                <string>Adjust Gaussian Filter Sigma</string>
               </property>
               <property name="maximum">
                <number>512</number>
               </property>
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_10">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_chiu">
      <attribute name="title">
       <string>Chiu</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_14">
       <item>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0" colspan="2">
          <widget class="QCheckBox" name="checkBox_chiuEnabled">
           <property name="toolTip">
            <string>Enable Chiu Noise Reduction Filter</string>
           </property>
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3" colspan="2">
          <widget class="QCheckBox" name="checkBox_includeCenter">
           <property name="toolTip">
            <string>Include Center Pixel in Filter</string>
           </property>
           <property name="text">
            <string>Include center</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_chiuRadius">
           <property name="text">
            <string>Radius</string>
           </property>
           <property name="buddy">
            <cstring>slider_chiuRadius</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1" colspan="3">
          <widget class="QSlider" name="slider_chiuRadius">
           <property name="toolTip">
            <string>Adjust Filter Radius</string>
           </property>
           <property name="maximum">
            <number>512</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QDoubleSpinBox" name="spinBox_chiuRadius">
           <property name="toolTip">
            <string>Adjust Filter Radius Value</string>
           </property>
           <property name="keyboardTracking">
            <bool>false</bool>
           </property>
           <property name="minimum">
            <double>1.000000000000000</double>
           </property>
           <property name="maximum">
            <double>9.000000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.100000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
